doc/primer.rst

```diff
@@ -248,19 +246,19 @@ For example we can create a circular face like so ::
   In CadQuery (and OCCT) all the topological classes are shapes, the :class:`~cadquery.Shape` class is the most abstract topological class. 
   The topological class inherits :class:`~cadquery.Mixin3D` or :class:`~cadquery.Mixin1D` which provide aditional methods that are shared between the classes that inherits them.
 
-The direct API as it's name suggest doesn't provide a parent/children data structure, each method call return directly an object of the specified topological type.
+The direct API as its name suggest doesn't provide a parent/children data structure, instead each method call directly returns an object of the specified topological type.
 It is more verbose than the fluent API and more tedious to work with, but as it offer more flexibility (you can work with faces, which is something you can't do in the fluent API)
 it is sometimes more convenient than the fluent API.
 
 The OCCT API
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Finally we are discussing about the OCCT API. The OCCT API is the lowest level of CadQuery. The direct API is build upon the OCCT API, the OCCT API in CadQuery is available through OCP.
-OCP are the Python bindings of the OCCT C++ library CadQuery uses. This means you have access to all the OCCT C++ libraries in Python and in CadQuery.
+Finally we are discussing about the OCCT API. The OCCT API is the lowest level of CadQuery. The direct API is built upon the OCCT API, where the OCCT API in CadQuery is available through OCP.
+OCP are the Python bindings of the OCCT C++ libraries CadQuery uses. This means you have access to all the OCCT C++ libraries in Python and in CadQuery.
```

Comment:
I don't think that all the OCCT C++ classes/methods are wrapped.

Reply:
Most of the classes...

Reply:
I thought they all were, maybe there is technical exceptions but since you have access to the classes of OCAF and 3D visualisation I assume it's safe to say it like that ?

Reply:
```suggestion
OCP are the Python bindings of the OCCT C++ libraries CadQuery uses. This means you have access to (almost) all the OCCT C++ libraries in Python and in CadQuery.
```